release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1209,6 +1209,10 @@ OpenShift SDN network plugin was deprecated in 4.15 and 4.16. With this release,
 [id="ocp-4-17-bare-metal-hardware-bug-fixes_{context}"]
 ==== Bare Metal Hardware Provisioning
 
+* Previously, attempting to configure RAID on specific hardware models by using Redfish might result in the following error: "The attribute StorageControllers/Name is missing from the resource." With this update, the validation logic no longer requires the `Name` field, because it is not mandated by the Redfish standard. (link:https://issues.redhat.com/browse/OCPBUGS-38465[*OCPBUGS-38465*])
+
+* Previously, the management interface for `idrac-redfish` in the `redfish’ Bare Metal Operator (BMO) module was incorrectly set to iPXE. This caused the error "Could not find the following interface in the 'ironic.hardware.interfaces.management' entrypoint: ipxe." and deployment failed on Dell Remote Access Controller (iDRAC)-based servers. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-37261[*OCPBUGS-37261*])
```

Comment:
> the management interface for `idrac-redfish`

Should this be _[iDRAC9 Redfish management interface](https://www.dell.com/support/manuals/en-us/idrac9-lifecycle-controller-v4.x-series/idrac9_4.00.00.00_redfishapiguide_pub/overview?guid=guid-e85fd9c0-f4d1-4eff-be5d-550ebb77ff0d&lang=en-us)_ for clarity? 

At least, it feels like you need a noun after `idrac-redfish`